lib/src/presentation/widgets/loaders/default_loader.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/material.dart';
+
+class DefaultLoader extends StatelessWidget{
```

Comment:
format

Reply:
What do you mean with format? I see it right